src/model/github.ts

```diff
@@ -127,7 +128,7 @@ class GitHub {
       data.conclusion = result;
     }
 
-    await (CloudRunner.isCloudRunnerAsyncEnvironment
+    await (CloudRunner.isCloudRunnerAsyncEnvironment || GitHub.asyncWorkflows
       ? GitHub.runUpdateAsyncChecksWorkflow(data, `update`)
       : GitHub.updateGitHubCheckRequest(data));
```

Comment:
Needlessly complex. Consider reducing nesting as much as useful

```ts
if (isAsync) {
  await something
  return
}

await other
```

Reply:
Ah, this is basically a test override to force async with a bad name. 

There are a few weird bits because the feature set had to change a bit to stabilize k8s. And now I'm ironing the weird stuff out. 

I've renamed this to `forceAsyncTest` for now, I'll consider what I need to do to consolidate that further.